honeybee_radiance/sensorgrid.py

```diff
@@ -190,6 +198,40 @@ def count(self):
         """Number of sensors."""
         return len(self._sensors)
 
+    @property
+    def room_identifier(self):
+        """Get or set text for the Room identifier to which this SensorGrid belongs.
+        
+        This will be used in the model_folder_dict method to narrow down the
+        number of aperture groups that have to be run with this sensor grid.
+        If None, the grid will be run with all aperture groups in the model.
+        """
+        return self._room_identifier
+
+    @room_identifier.setter
+    def room_identifier(self, n):
+        self._room_identifier = typing.valid_string(n)
+
+    def model_folder_dict(self, model=None):
```

Comment:
I also think we should have an option to assign aperture group names directly.

The current method will work with the set-up of generating single sensor grid for a room perfectly but there can be other cases. Imagine an L shape room with two windows at the end. One wing will be mapped to one aperture group and the other wing will be a separate sensor-grid that will be mapped to the other one.

Reply:
Ok, I'll add an input for it but, given the only thing that this method does right now is auto-calculate the light path and write the number of points, adding an option to specify the light path is kinda a case where the input and output are really similar. I guess if we start adding more information into this dictionary, this will make more sense.

Reply:
I think this is part of the longer conversation of if we keep the `room_identifier` or the `aperture_group_identifiers`. See below.

I reviewed the code form bottom to top and the comments are reversed! You should probably start from the last comment and make it to the top. Sorry!

Reply:
Done.